src/languages/es.js

```diff
@@ -865,7 +865,7 @@ export default {
             pleaseSelectUser: 'Asegúrese de que el correo electrónico o el número de teléfono sean válidos (p. ej. +15005550006).',
```

Comment:
I don't know why this message(pleaseSelectUser) is different in spanish, the english one doesn't have a number and says `Please select a user from contacts.`

Also I think we don't need it now, this error message is for a case when a user tries to invite without selecting a user but this case isn't possible because the invite button stays disabled. Also it looks like we aren't using it (getErrorText isn't being used)
https://github.com/Expensify/App/blob/43933e38a73d6703f0b9764007038fa530fe1d42/src/pages/workspace/WorkspaceInvitePage.js#L109-L114

Reply:
Should I remove getErrorText ?